pkg/endpoint/local_endpoint.go

```diff
@@ -171,42 +174,67 @@ func GetLocalSpec(ctx context.Context, submSpec *types.SubmarinerSpecification,
 	}
 
 	endpointSpec := &submv1.EndpointSpec{
-		CableName:     fmt.Sprintf("submariner-cable-%s-%s", submSpec.ClusterID, strings.ReplaceAll(privateIP, ".", "-")),
 		ClusterID:     submSpec.ClusterID,
 		Hostname:      hostname,
-		PrivateIP:     privateIP,
 		NATEnabled:    submSpec.NATEnabled,
 		Subnets:       localSubnets,
 		Backend:       submSpec.CableDriver,
 		BackendConfig: backendConfig,
 	}
 
-	publicIP, resolver, err := getPublicIP(submSpec, k8sClient, backendConfig, airGappedDeployment)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not determine public IP")
+	for _, privateIP := range privateIPs {
+		endpointSpec.SetPrivateIP(privateIP)
```

Comment:
Instead if `GetLocalIPs` above, we should simplify and pass in the family to `GetLocalIP`. 

```suggestion
        for _, family := range submSpec.GetIPFamilies() {
            endpointSpec.SetPrivateIP(GetLocalIP(family))
        }
```